node/test_force.js

```diff
@@ -286,6 +287,16 @@ function editForceScriptToUseLocalPluginRepo(tmpDir, os) {
 }
 
 //
+// Update createApp.sh to not do pod install
+// 
+function editCreateAppToNotDoPodInstall(tmpDir) {
+    log('Editing  createApp.sh to not do pod install', COLOR.green);
+    miscUtils.replaceTextInFile(path.join(tmpDir, 'node_modules', 'forceios', 'build', 'app_template_files', 'createApp.sh'), new RegExp('pod install$', 'g'), '# pod install');
```

Comment:
Done here: https://github.com/forcedotcom/SalesforceMobileSDK-Shared/pull/408/commits/50e4233facf22fdc8baca49d1118611e7ddf6413

NB: 
it has to be either
new RegExp(/^\s*pod install/, 'g')
or
new RegExp('^\\s*pod install', 'g')